README.md

```diff
@@ -51,7 +51,6 @@ The `Maintained` column shows:
 <!-- Please keep this list sorted in alphabetic order -->
 | Platform | Composite PLATFORM flag | Publicly available? | Maintained? |
 |----------|-------------------------|---------------------|-------------|
-| [Allwinner A80 Board](http://linux-sunxi.org/A80)|`PLATFORM=sunxi`| No | ![Not maintained](documentation/images/red.svg) v0.1.0 |
```

Comment:
Everything in this PR seems fine to me, I just wonder if we should keep this particular line. I.e., if for some reason people want to go back and use old platforms not supported any longer, they will at least know where to start from.

Reply:
If we want to keep this line, we would need to update the description of the table, because currently it is supposed to contain only supported platforms, the red image just means it's been untested for some time.

Reply:
Maybe just as simple as adding another column like `"comment"` where we say that this is no longer in the (current) repository. Alternatively something like 
```
| [Allwinner A80 Board](http://linux-sunxi.org/A80)|`PLATFORM=sunxi`| No | ![Not maintained](documentation/images/red.svg) v0.1.0 (platform removed from tip) |
```
Drawback with the latter is that it widens the column a lot.

Reply:
I think it might be a bit confusing and not really worth the trouble.

Reply:
@jforissier, agree.

Reply:
Alright, let's just remove it and then we can see in the future if / when there something else we can come up with if there are other obsolete platforms.